### PR TITLE
Init Model

### PR DESCRIPTION
**Pull request by YenNguyenBKHN**
==> Title: Init Model

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/19385

==> Attachment File:
Screenshot from 2019-11-15 10-07-28

### CODE REVIEW

**Comment by YenNguyenBKHN** [comment]
ready

**Comment by phucmh-1356** [comment]
commented
ignore folder log đi em

**Comment by YenNguyenBKHN** [comment]
ready

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :commentable, polymorphic: true
+  belongs_to :patient
+  belongs_to :new
+  belongs_to :doctor
```
Comment (manhbnt-0960) [inline app/models/comment.rb]: gì thế e ?

**Comment by YenNguyenBKHN** [comment]
ready

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  self.inheritance_column = :role
+
+  def self.roles
+    %w(Doctor Patient Staff)
+  end
```
Comment (phucmh-1356) [inline app/models/user.rb]: dùng class << self đi em

**Comment by YenNguyenBKHN** [comment]
ready

**Pull request merged**